build.sh

```diff
@@ -7,10 +7,10 @@ extra_flags=''
 clean="clean"
 while getopts 'fd' flag; do
   case "${flag}" in
-    f) extra_flags='-Drascal.compile.skip -Drascal.tutor.skip' ;;
+    f) extra_flags='-Drascal.compile.skip -Drascal.tutor.skip -DskipTests' ;;
```

Comment:
Are these changes committed intentionally?

Reply:
yes, but primarily since I didn't want to have a separate PR just for that addition.